Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -7175,7 +7195,71 @@ public void SteamStopGearBoxDecrease()
 
         }
 
-        //Small Ejector Controller
+        #region Steam booster controller
+
+        public void StartSteamBoosterIncrease(float? target)
+        {
+            SteamBoosterController.CommandStartTime = Simulator.ClockTime;
+            if (IsPlayerTrain)
+                Simulator.Confirmer.ConfirmWithPerCent(CabControl.SteamBooster, CabSetting.Increase, SteamBoosterController.CurrentValue* 100);
```

Comment:
```suggestion
                Simulator.Confirmer.ConfirmWithPerCent(CabControl.SteamBooster, CabSetting.Increase, SteamBoosterController.CurrentValue * 100);
```

Reply:
Thanks, I have made this adjustment, and also to a number of other similar entries.